src/applications/disability-benefits/526EZ/manifest.json

```diff
@@ -3,5 +3,22 @@
   "entryFile": "./form-entry.jsx",
   "entryName": "526EZ-claims-increase",
   "rootUrl": "/disability-benefits/apply/form-526-disability-claim",
-  "production": true
+  "template": {
```

Comment:
Is this describing a template or are these values pushed into a template- like a context in handlebars?

Reply:
It's pushed into a template-like context. Everything under `template` is the same as frontmatter in Markdown files. Properties outside of `template` like `entryFile` are meaningful to the manifest-loader/Webpack.